packages/kbn-ui-shared-deps/scripts/build.js

```diff
@@ -57,9 +55,9 @@ run(
             },
           ];
 
-          log.debug('metrics:', metrics);
-
-          await reporter.metrics(metrics);
+          const metricsPath = Path.resolve(DIST_DIR, 'metrics.json');
```

Comment:
Can we move all of that logic inside the webpack config for that package itself (like an inplace webpack plugin) as we are going to remove that build file as part of the Bazel work?

Reply:
Sure, that's what I did for the optimizer